CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ Table of Contents
 
 ### Improvements
 
+* Adding min-height, min-width and vertical-align to positioning.filter, as these govern element positions.
```

Comment:
Thanks for updating the changelog. However, I recall that we agreed that for these changes we do not update the changelog.